tests/safeds/data/tabular/transformation/test_discretizer.py

```diff
@@ -0,0 +1,223 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation import Discretizer
+from safeds.exceptions import TransformerNotFittedError, UnknownColumnNameError
+
+
+class TestInit:
+    def test_should_raise_value_error(self) -> None:
+        with pytest.raises(ValueError, match="Parameter 'number_of_bins' must be >= 2."):
+            _ = Discretizer(1)
+
+
+class TestFit:
+    def test_should_raise_if_column_not_found(self) -> None:
+        table = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        with pytest.raises(UnknownColumnNameError):
+            Discretizer().fit(table, ["col2"])
+
+    def test_should_not_change_original_transformer(self) -> None:
+        table = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        transformer = Discretizer()
+        transformer.fit(table, None)
+
+        assert transformer._wrapped_transformer is None
+        assert transformer._column_names is None
+
+
+class TestTransform:
+    def test_should_raise_if_column_not_found(self) -> None:
+        table_to_fit = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        transformer = Discretizer().fit(table_to_fit, None)
+
+        table_to_transform = Table(
+            {
+                "col2": ["a", "b", "c"],
+            },
+        )
+
+        with pytest.raises(UnknownColumnNameError):
```

Comment:
Please test for the specific error message with the `match` parameter in `pytest.raises`